docs/git-compatibility.md

```diff
@@ -109,6 +93,59 @@ https://github.com/octocat/Hello-World` will clone GitHub's "Hello-World" repo
 into a directory by the same name.
 
 
+## Co-located Jujutsu/Git repos
+
+A "co-located" Jujutsu repo is a hybrid Jujutsu/Git repo. These can be created
+with `jj git clone --colocate` or if you initialize the Jujutsu repo in an
+existing Git repo by running `jj init --git-repo=.`. The Git repo and the
+Jujutsu repo then share the same working copy. Jujutsu will import and export
+from and to the Git repo on every `jj` command automatically.
+
+This mode is very convenient useful when tools (e.g. build tools) expect a Git
+repo to be present.
+
+It is allowed to mix `jj` and `git` commands in such a repo in any order.
+However, it may be easier to keep track of what is going on if you generally use
+read-only `git` commands and use `jj` for any commands that mutate the repo, or
+vice-versa.
+
+You can even undo the results of mutating git commands using `jj undo` and `jj
+op restore`. Inside `jj op log`, changes by `git` will be represented as a
+"import git refs" operation.
+
+There are a few downsides to this mode of operation. Generally, using colocated
+repos may require you to deal with more involved Jujutsu and Git concepts.
+
+* Interleaving `jj` and `git` commands increases the chance of confusing branch
+  conflicts or conflicted (AKA divergent) change ids. These never lose data, but
+  can be annoying.
+
+* Git tools will have trouble with revisions that contain conflicted files. While
+  `jj` renders these files with conflict markers in the working copy, they are
+  stored in a non-human-readable fashion inside the repo. Git tools will often
+  see this non-human-readable representations.
+
+* When a `jj` branch is conflicted, the position of the branch in the Git repo
+  will disagree with one or more of the conflicted positions. The state of that
+  branch in git will be labeled as though it belongs to a remote named "git",
+  e.g. `branch@git`.
+
+* Jujutsu will ignore Git's staging area. It will not understand merge conflicts
+  as Git represents them, unfinished `git rebase` states, as well as other less common
+  states a Git repository can be in.
```

Comment:
Is this necessary?

Reply:
What do you mean? Do you think this is too obvious to say?

Reply:
Yes, it's obvious. It could be rephrased with something like: "As Jujutsu is only aware of finished Git operations [...]", but I don't have an idea on how finish the sentence.

Reply:
I think this is the kind of thing that might be obvious once you are familiar enough with how `jj` works, but might not be obvious to beginners. So, I'm think of keeping this, unless I can think of a better way to communicate it.